packages/material-ui-system/src/index.d.ts

```diff
@@ -82,69 +79,6 @@ export const sizeHeight: SimpleStyleFunction<'sizeHeight'>;
 export const boxSizing: SimpleStyleFunction<'boxSizing'>;
 export type SizingProps = PropsFor<typeof sizing>;
 
-// spacing.js
```

Comment:
Moved to `spacing.d.ts`

Reply:
Oh yes, we need to breakdown this file 😅